StreamChat_v3.xcodeproj/project.pbxproj

```diff
@@ -2453,6 +2553,42 @@
 			productReference = 792DDA57256FB69E001DB91B /* DemoApp.app */;
 			productType = "com.apple.product-type.application";
 		};
+		793060E525778896005CF846 /* StreamChatTestTools */ = {
+			isa = PBXNativeTarget;
+			buildConfigurationList = 793060FD25778897005CF846 /* Build configuration list for PBXNativeTarget "StreamChatTestTools" */;
+			buildPhases = (
+				793060E125778896005CF846 /* Headers */,
+				793060E225778896005CF846 /* Sources */,
+				793060E325778896005CF846 /* Frameworks */,
+				793060E425778896005CF846 /* Resources */,
+			);
+			buildRules = (
+			);
+			dependencies = (
+			);
+			name = StreamChatTestTools;
+			productName = StreamChatTestTools;
+			productReference = 793060E625778896005CF846 /* StreamChatTestTools.framework */;
+			productType = "com.apple.product-type.framework";
+		};
+		793060ED25778897005CF846 /* StreamChatTestToolsTests */ = {
+			isa = PBXNativeTarget;
+			buildConfigurationList = 793060FE25778897005CF846 /* Build configuration list for PBXNativeTarget "StreamChatTestToolsTests" */;
+			buildPhases = (
+				793060EA25778897005CF846 /* Sources */,
+				793060EB25778897005CF846 /* Frameworks */,
+				793060EC25778897005CF846 /* Resources */,
+			);
+			buildRules = (
+			);
+			dependencies = (
+				793060F125778897005CF846 /* PBXTargetDependency */,
+			);
+			name = StreamChatTestToolsTests;
```

Comment:
nit: I really do not understand what's wrong with `StreamCounterintelligenceService` name :(

Reply:
The world is not ready yet...